arlas-hits-exporter/Docker/export_data.bash

```diff
@@ -40,24 +39,22 @@ case "$i" in
     ;;
 esac
 done
-echo $SEARCH_URL
-if [ ! -v SEARCH_URL ]; then echo "--search_url argument is missing"; usage; else echo "### Your search request is ==> ${SEARCH_URL}"; fi
-if [ ! -v INCLUDE ]; then echo "--include argument is missing" & usage; else echo "### Fields to export ==> ${INCLUDE}"; fi
-if [ ! -v OUTPUT ]; then OUTPUT=$PWD; echo "### Output file ==> ${OUTPUT}" else echo "### Output file ==> ${OUTPUT}"; fi
+if [ ! -v SEARCH_URL ]; then echo "ERROR : --search_url argument is missing"; usage; else echo "### Your search request is ==> ${SEARCH_URL}"; fi
```

Comment:
Shouldn't the script exit with an error code (`exit 1`) if the variable is undefined?

Also, error messages should be written to standard error (file descriptor `2`), instead of standard output (default, file descriptor `1`):

```bash
>&2 echo "error message"
```

Reply:
The `exit 1` error code is called in usage function